distribution/pom.xml

```diff
@@ -242,6 +242,58 @@
 
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <outputDirectory>target</outputDirectory>
+                    <thirdPartyFilename>NOTICE.txt</thirdPartyFilename>
+                    <acceptPomPackaging>true</acceptPomPackaging>
+                    <fileTemplate>org/kaazing/community/notice/third-party-template.ftl</fileTemplate>
+                    <useMissingFile>true</useMissingFile>
+                    <missingFile>src/license/THIRD-PARTY.properties</missingFile>
+                    <failIfWarning>true</failIfWarning>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>add-third-party</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                   </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.kaazing</groupId>
+                        <artifactId>community.enforcer</artifactId>
+                        <version>develop-SNAPSHOT</version>
```

Comment:
develop-SNAPSHOT here seems like this would eventually break when released.  Maybe you want ${project.version}?   I'm not sure what would happen when it is released if this property would be overridden or it would stay project.version.  So down stream builds of say kaazing/gateway might be looking for their project.verison???